kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/tasks/services/KoverArtifactGenerationTask.kt

```diff
@@ -19,7 +19,6 @@ import javax.inject.*
  *
  * This artifact that will be shared between projects through dependencies for creating merged reports.
  */
-@CacheableTask
```

Comment:
Also, please, rerun all tests after this change, `BuildCacheRelocationTests` must fall.